Adyen/Core/Core Protocols/PaymentMethod.swift

```diff
@@ -15,27 +15,39 @@ public protocol PaymentMethod: Decodable {
     /// The name of the payment method, such as `"Credit Card"`, `"iDEAL"`, `"Apple Pay"`.
     var name: String { get }
     
-    /// Display information for the payment method, adapted for displaying in a list.
-    var displayInformation: DisplayInformation { get }
+    /// Describes a payment method display information thats provided by the merchant
+    /// and if not `nil`, will override the display information generated by ``defaultDisplayInformation(using:)``.
+    var merchantProvidedDisplayInformation: MerchantCustomDisplayInformation? { get set }
     
     /// Display information for the payment method, adapted for displaying in a list.
     ///
     /// - Parameters:
     ///   - using: The localization parameters.
-    func localizedDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation
+    /// :nodoc:
+    func defaultDisplayInformation(using parameters: LocalizationParameters?) -> DisplayInformation
```

Comment:
Why this one is `/// :nodoc:` ?

Reply:
because you shouldn't call this one directly, alway throw the `displayInformation()` function to use the merchant provided display info if its set, so want this one to be only overriden by `PaymentMethod` implementations if needed.

Reply:
ok, make sense!